config/static.go

```diff
@@ -118,6 +116,11 @@ func loadStaticConfig(cfgPath string) (*StaticCfg, error) {
 	// set the socket time out in hours
 	config.MongoDB.SocketTimeout *= time.Hour
 
+	// clean all filepaths
+	config.Log.RitaLogPath = filepath.Clean(config.Log.RitaLogPath)
+	config.Blacklisted.SafeBrowsing.Database = filepath.Clean(config.Blacklisted.SafeBrowsing.Database)
+	config.Bro.ImportDirectory = filepath.Clean(config.Bro.ImportDirectory)
```

Comment:
`filepath.Clean` seems to be a good one to use here. It removes the trailing slash and takes care of some other possible filepath cruft. I went ahead and ran it on the other config values that contain filepaths.